packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -59,6 +59,8 @@ import {inspectHooksOfFiber} from 'react-debug-tools';
 import {
   patch as patchConsole,
   registerRenderer as registerRendererWithConsole,
+  patchForStrictMode,
+  unpatchForStrictMode,
```

Comment:
nit: I think you could just import these _as_ and export them directly (and avoid the wrapper functions). The only reason I mention this is that the name "patchForStrictMode" isn't the most meaningful (without the "console" part).
```js
import {
  patch as patchConsole,
  registerRenderer as registerRendererWithConsole,
  patchForStrictMode as patchConsoleForStrictMode,
  unpatchForStrictMode as unpatchConsoleForStrictMode,
} from './console';

// ...

return {
  patchConsoleForStrictMode,
  unpatchConsoleForStrictMode,
  // ...
};
```